README.md

```diff
@@ -1 +1,12 @@
+[![Build Status](https://travis-ci.org/andela-gike/Checkpoint-1.svg?branch=master)](https://travis-ci.org/andela-gike/Checkpoint-1)
+[![Coverage Status](https://coveralls.io/repos/github/andela-gike/Checkpoint-1/badge.svg?branch=master)](https://coveralls.io/github/andela-gike/Checkpoint-1?branch=master)
+[![Issue Count](https://codeclimate.com/github/andela-gike/Checkpoint-1/badges/issue_count.svg)](https://codeclimate.com/github/andela-gike/Checkpoint-1)
+[![Test Coverage](https://codeclimate.com/github/andela-gike/Checkpoint-1/badges/coverage.svg)](https://codeclimate.com/github/andela-gike/Checkpoint-1/coverage)
+[![Code Climate](https://codeclimate.com/github/andela-gike/Checkpoint-1/badges/gpa.svg)](https://codeclimate.com/github/andela-gike/Checkpoint-1)
+
 # inverted-index
+// if (Object.keys(this.index) === 0) {
+    //   return false;
+    // } else if (typeof filePath !== 'string' || typeof filePath === 'undefined') {
+    //   return this.index;
+    // }
```

Comment:
You have commented code in your README. I'm sure this is an unintended error; you should remove it.